### ISSUE TITLE
insecure instructions

### ISSUE THREAD

**Issue by bozo32** [JSv4/Delphic#39]
the installation instructions suggest sudo
this seems not to be necessary.

**Comment by JSv4** [comment]
Fair enough. Depends on how you setup docker. If you want to update the instructions I'd definitely welcome a PR.
